kafka/util.py

```diff
@@ -95,6 +95,8 @@ def kafka_bytestring(s):
         return s
     if isinstance(s, six.string_types):
         return s.encode('utf-8')
+    if s is None:
```

Comment:
group was None in some circumstances. Can catch that before calling kafka_bytestring(group).